### PR TITLE
APP-2054: Update view-manage-entities.md

### PR DESCRIPTION
**Pull request by davidsnyder-jask**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [X] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cse/records-signals-entities-insights/view-manage-entities.md [hunk 1]
```diff
@@ -171,16 +171,16 @@ Note that:
 id,suppressed,criticality,tags,tags_to_add,tags_to_remove
 _ip-zone1-10.0.0.5,false,default,,Office-Based,
 _ip-zone1-10.0.0.6,true,default,,Office-Based,Remote
-_ip-zone1-10.0.0.7,false,Executive,Office-Based,,
```
Comment (davidsnyder-jask) [inline docs/cse/records-signals-entities-insights/view-manage-entities.md:174]: this didn't follow our instructions a few lines up of not including `tags` when you have `tags_to_add`: https://github.com/SumoLogic/sumologic-documentation/compare/main...davidsnyder-jask:sumologic-documentation:patch-2#diff-aba5be95cb80648467ad531397991d4df3d4843dc44d35824977e922d943e10eL152

File: docs/cse/records-signals-entities-insights/view-manage-entities.md [hunk 2]
```diff
@@ -171,16 +171,16 @@ Note that:
...
 ```
 
 **Format 2 example**
 
 ```
 type,value,sensor_zone,suppressed,criticality,tags,tags_to_add,tags_to_remove
-_ip,10.0.0.5,zone1,false,Executive PC,Frequent-Travel,,
+_ip,10.0.0.5,zone1,false,default,Frequent-Travel,,
 _ip,10.0.0.6,zone1,true,default,,Office-Based,Remote
-_ip,10.0.0.7,zone1,false,Executive,Office-Based,
```
Comment (davidsnyder-jask) [inline docs/cse/records-signals-entities-insights/view-manage-entities.md:183]: same as above, but these were missing the `criticality` column

**Review by MariePMcGarry — APPROVED** [review]

File: docs/cse/records-signals-entities-insights/view-manage-entities.md [hunk 3]
```diff
@@ -171,16 +171,16 @@ Note that:
 id,suppressed,criticality,tags,tags_to_add,tags_to_remove
 _ip-zone1-10.0.0.5,false,default,,Office-Based,
 _ip-zone1-10.0.0.6,true,default,,Office-Based,Remote
-_ip-zone1-10.0.0.7,false,Executive,Office-Based,,
+_ip-zone1-10.0.0.7,false,default,,Office-Based,
```
Comment (kimsauce) [inline docs/cse/records-signals-entities-insights/view-manage-entities.md:174]:
```suggestion
_ip-zone1-10.0.0.7,false,default,Office-Based,
```

File: docs/cse/records-signals-entities-insights/view-manage-entities.md [hunk 4]
```diff
@@ -171,16 +171,16 @@ Note that:
 id,suppressed,criticality,tags,tags_to_add,tags_to_remove
 _ip-zone1-10.0.0.5,false,default,,Office-Based,
 _ip-zone1-10.0.0.6,true,default,,Office-Based,Remote
```
Comment (kimsauce) [inline docs/cse/records-signals-entities-insights/view-manage-entities.md:173]: @davidsnyder-jask @MariePMcGarry are the extra commas intentional?

Comment (MariePMcGarry) [reply docs/cse/records-signals-entities-insights/view-manage-entities.md:173]: yes, @kimsauce. That's how you supply a blank value.

**Pull request merged**